kairos-cli/src/crypto/signer.rs

```diff
@@ -0,0 +1,38 @@
+use std::path::Path;
+
+use super::private_key::CasperPrivateKey;
+use super::public_key::CasperPublicKey;
+use crate::crypto::error::CryptoError;
+use casper_types::bytesrepr::ToBytes;
+use casper_types::{crypto, ErrorExt, PublicKey};
+
+pub struct CasperSigner {
+    secret_key: CasperPrivateKey,
+    pub public_key: CasperPublicKey,
+}
+
+#[allow(unused)]
```

Comment:
Isn't this used though?

Reply:
Not used, because we have no signature verification yet:

https://github.com/cspr-rad/kairos/blob/2ef83df22bbdfa06f2d49b8fed309c0274ba907d/kairos-server/src/routes/transfer.rs#L34

This is about to come soon - once we establish transaction format (I am currently exploring protobuffs and ASN.1+DER).